frontend/js/components/VSelect.vue

```diff
@@ -168,7 +168,16 @@
             if (this.taggable) {
               this.value = value
             } else {
-              this.value = this.options.filter(o => value.includes(o.value))
```

Comment:
@ifox @antonioribeiro 

I had to add this chunk of js to ensure that the options are set to the order of how the id's come in. I believe this also needs to be in 2.x as otherwise it will take the order of all the available options (which is not sorted)

Reply:
@antonioribeiro can you share more about your use case for storing the position on multiselect? How are you using the field: which options do you use and how are the values loaded?

Reply:
ping @antonioribeiro

Reply:
Hey @ifox, use case is basically when we need to read the selected items in the order we are selecting them.

On the screenshot below I selected "Education" first, because I want Education to be shown as the main sector in my blog post page.

<img width="1479" alt="Screenshot 2024-01-26 at 15 16 29" src="https://github.com/area17/twill/assets/3182864/b56b7e4e-d4e1-402c-b07b-b9a39a1fe36f">

When I save and reload the model, Education is no more the first item and the frontend will show "Housing" as the main sector:

<img width="1479" alt="Screenshot 2024-01-26 at 15 14 26" src="https://github.com/area17/twill/assets/3182864/8ac56d06-be50-48cb-a11d-fc500ee577a2">

It doesn't really matter how I'm getting these records later because their positions are all `null` on the relationship table:

<img width="670" alt="Screenshot 2024-01-26 at 15 26 25" src="https://github.com/area17/twill/assets/3182864/55d91f31-4e0d-49ce-abda-af8edd043ba3">

There will be now way for me to read them in the right order.

Reply:
This is how I'm creating the form field to test on Twill 3, where I get the same behaviour:

```php
$form->add(
    MultiSelect::make()
        ->name("sectors")
        ->inline()
        ->options(
            Options::make(
                Sector::all()
                    ->map(
                        fn($sector) => Option::make(
                            $sector->id,
                            $sector->title
                        )
                    )
                    ->toArray()
            )
        )
);

```